pkgs/games/anki/bin.nix

```
@@ -0,0 +1,67 @@
+{ fetchurl, stdenv, lib, buildFHSUserEnv, appimageTools, writeShellScript, anki, undmg }:
+
+let
+  pname = "anki-bin";
+  version = "2.1.36";
+
+  tarball = fetchurl {
+    url = "https://github.com/ankitects/anki/releases/download/${version}/anki-${version}-linux.tar.bz2";
+    sha256 = "01xcjnfs5pfh7v0nkffw2wpl19l6pj9k3kxrcawv3cm42asy0mfz";
+  };
+
+  unpacked = stdenv.mkDerivation {
+    inherit pname version;
+
+    src = tarball;
```

Might as well inline
```suggestion
    src = fetchurl {
      url = "https://github.com/ankitects/anki/releases/download/${version}/anki-${version}-linux.tar.bz2";
      sha256 = "01xcjnfs5pfh7v0nkffw2wpl19l6pj9k3kxrcawv3cm42asy0mfz";
    };
```
same goes for `dmg`